src/cmd/cli/command/compose.go

```diff
@@ -45,6 +67,11 @@ func makeComposeUpCmd() *cobra.Command {
 
 			printPlaygroundPortalServiceURLs(deploy.Services)
 
+			var hasOnlyManagedResources = IsOnlyManagedResources(project)
```

Comment:
naming: there's no reason to go from "is" to "has". Pick one. This helps searching all occurrences and uses of it.